src/block_cache/block_chunkcache.c

```diff
@@ -28,28 +65,35 @@ static int
 __chunkcache_alloc(WT_SESSION_IMPL *session, WT_CHUNKCACHE_CHUNK *chunk)
 {
     WT_CHUNKCACHE *chunkcache;
-    WT_DECL_RET;
+    size_t bit_index;
+    uint8_t *map_byte;
 
     chunkcache = &S2C(session)->chunkcache;
+    bit_index = 0;
 
     if (chunkcache->type == WT_CHUNKCACHE_IN_VOLATILE_MEMORY)
-        ret = __wt_malloc(session, chunk->chunk_size, &chunk->chunk_memory);
+        WT_RET(__wt_malloc(session, chunk->chunk_size, &chunk->chunk_memory));
     else {
-#ifdef ENABLE_MEMKIND
-        chunk->chunk_memory = memkind_malloc(chunkcache->memkind, chunk->chunk_size);
+retry:
+        /* Use the bitmap to find the free chunks in the cache. */
+        WT_RET(__bitmap_find_free(session, &bit_index));
```

Comment:
Should encapsulate `__bitmap_find_free` with `WT_RET_MSG` to add an error message?

Reply:
@Siddhartha8899  I think it is best to leave the current error message inside - as we could fail in multiple parts of bitmap_find_free so it would be misleading to place that current message out here. I think WT_RET would suffice, in saying we have a failure inside bitmap_find_free as theres no other general error message I would write here.